### PR TITLE
downloadIsisData Script fixes

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

Fixes random bugs that ended up in the new downloadIsisData script. Also patches the README to remove all mentions to .py that also somehow got through. 

## Related Issue

https://github.com/USGS-Astrogeology/ISIS3/issues/5024

## How Has This Been Tested?

Local install and this is the version running nightly on the isismgr_scripts repo

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/scripts/downloadIsisData [hunk 1]
```diff
@@ -16,17 +15,20 @@ from os import path
...
-    install_path = Path(os.environ.get("CONDA_PREFIX")) / "etc" / "isis" / local_path
-    repo_path = Path(os.path.dirname(__file__)) / '..' / 'config' / 'rclone.conf' 
-    
+    # this should be installed in scripts folder, so config is one directory up in etc 
+    install_path = Path(os.path.dirname(__file__)) / '..' / "etc" / "isis" / 'rclone.conf'
+    repo_path = Path(os.path.dirname(__file__)) / '..' / 'config' / 'rclone.conf'
+    print(install_path)    
     if local_path.exists():
-        return str(local_path) 
+        print(str(local_path))
```
Comment (jlaura) [inline isis/scripts/downloadIsisData]: Is this print intentional? Swap for a logging statement?

File: isis/scripts/downloadIsisData [hunk 2]
```diff
@@ -16,17 +15,20 @@ from os import path
...
+        return str(local_path)
     elif repo_path.exists():
         return str(repo_path)
     elif install_path.exists():
+        print(str(install_path))
```
Comment (jlaura) [inline isis/scripts/downloadIsisData]: Same print question.

**Comment by acpaquette** [comment]
@jlaura Is this good to go now?

**Comment by jlaura** [comment]
This has issues that are fixed in #5032. We could merge this and then ask @foobarbecue to rebase and then merge that. Or we could incorporate this changes into this PR and close #5032. I think I'd prefer to merge this, have #5032 rebase, and then merge that PR so that we have a record of the contributions.

**Comment by Kelvinrr** [comment]
@jlaura I agree with asking #5032 to rebase, we should get this merged first.

**Review by jlaura — APPROVED** [review]

**Comment by foobarbecue** [comment]
Gotcha, I'll rebase #5032 tomorrow

**Comment by foobarbecue** [comment]
BTW this was all very thoughtful but in the future no need to go out of the way to preserve my contributions especially if it's something this tiny!

**Pull request merged**